README.md

```diff
@@ -16,6 +16,11 @@ This repository holds the kcp tests that tests against the publicly available  i
   ```
 
 * Have the environment variable `KUBECONFIG` set pointing to your kcp service
+  - If your `KUBECONFIG` has multiple contexts please specific target test environment by using environment variable `E2E_TEST_CONTEXT` ( In addition, if the specified test context doesn't exist, it will try to use `kcp-stable-root` context, if `kcp-stable-root` doesn't exist either, it'll use the `current-context` in `KUBECONFIG` instead). 
+    ```console
```

Comment:
Reading the code, if E2E_TEST_CONTEXT is not set, it will use kcp-stable-root and if the specified test context does not exist it will use current-context. AFAIU if specified test context does not exist it will not use kcp-stable-root i believe.